README.rst

```diff
@@ -1,45 +1,48 @@
-|Codecov| |license|
-
-.. |codecov| image:: https://codecov.io/gh/edx/frontend-app-learning/branch/master/graph/badge.svg?token=3z7XvuzTq3
-   :target: https://codecov.io/gh/edx/frontend-app-communications
-.. |license| image:: https://img.shields.io/badge/license-AGPL-informational
-   :target: https://github.com/edx/frontend-app-account/blob/master/LICENSE
-
 frontend-app-communications
-==============================
+#############################
 
-Please tag **edx-aperture** on any PRs or issues.  Thanks!
+|license-badge| |status-badge| |ci-badge| |codecov-badge|
 
-Introduction
-------------
 
-A tool used by course teams to communicate with thier learners. The interface for anything related to instructor to learner communications. Instructor bulk email, for example.
+Purpose
+*******
 
+A tool used by course teams to communicate with thier learners. The interface for anything related to instructor to learner communications. Instructor bulk email, for example.
```

Comment:
I know this isn't your fault, but since you're in here -- could you fix the spelling of `thier` to `their`? Thanks in advance! :)

Reply:
No problem, thanks :)